<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BandWidget</class>
 <widget class="QWidget" name="BandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Band</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Band</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <property name="spacing">
       <number>1</number>
      </property>
      <item row="2" column="3">
       <widget class="AeroButton" name="bnd_70cm">
        <property name="text">
         <string>70 cm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="AeroButton" name="bnd_20m">
        <property name="text">
         <string>20 m</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="AeroButton" name="bnd_40m">
        <property name="text">
         <string>40 m</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="AeroButton" name="bnd_630m">
        <property name="text">
         <string>630 m</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="6">
       <widget class="AeroButton" name="bnd_gen">
        <property name="text">
         <string>Gen</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="AeroButton" name="bnd_17m">
        <property name="text">
         <string>17 m</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="AeroButton" name="bnd_2m">
        <property name="text">
         <string>2 m</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="AeroButton" name="bnd_12m">
        <property name="text">
         <string>12 m</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="AeroButton" name="bnd_10m">
        <property name="text">
         <string>10 m</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="AeroButton" name="bnd_30m">
        <property name="text">
         <string>30 m</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="AeroButton" name="bnd_23cm">
        <property name="text">
         <string>23 cm</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="AeroButton" name="bnd_6m">
        <property name="text">
         <string>6 m</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="AeroButton" name="bnd_15m">
        <property name="text">
         <string>15 m</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="AeroButton" name="bnd_33cm">
        <property name="text">
         <string>33 cm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="AeroButton" name="bnd_60m">
        <property name="text">
         <string>60 m</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="AeroButton" name="bnd_125cm">
        <property name="text">
         <string>125 cm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="AeroButton" name="bnd_80m">
        <property name="text">
         <string>80 m</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="AeroButton" name="bnd_160m">
        <property name="text">
         <string>160 m</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="AeroButton" name="bnd_2200m">
        <property name="text">
         <string>2200 m</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="AeroButton" name="bnd_13cm">
        <property name="text">
         <string>13 cm</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="AeroButton" name="bnd_10cm">
        <property name="text">
         <string>10 cm</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="AeroButton" name="bnd_5cm">
        <property name="text">
         <string>5 cm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AeroButton</class>
   <extends>QPushButton</extends>
   <header location="global">Util/cusdr_buttons.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
